pkg/platform/model/inventory.go

```diff
@@ -56,12 +55,15 @@ func IngredientByNameAndVersion(language, name, version string) (*IngredientAndV
 		if ingredient.Ingredient.Name == nil || *ingredient.Ingredient.Name != name {
 			continue
 		}
-		v := ingredient.Version.Version
-		if v != nil && *v == version {
-			return &IngredientAndVersion{
-				ingredient.V1IngredientAndVersion,
-				ingredient.Namespace,
-			}, nil
+
+		for _, ver := range ingredient.Versions {
+			v := ver.Version
+			if v == version {
+				return &IngredientAndVersion{
+					ingredient.V1SearchIngredientsResponseIngredientsItems, v,
```

Comment:
Nit: I was wondering where the version was until I saw it hiding on this line.

Reply:
I will make this more obvious. :)